packages/login_client/lib/src/login_client.dart

```diff
@@ -87,9 +88,21 @@ class LoginClient extends http.BaseClient {
   Future<oauth2.Credentials?> get credentials => _credentialsStorage.read();
 
   /// Restores saved credentials from the credentials storage.
-  Future<void> initialize() async {
-    final credentials = await _credentialsStorage.read();
+  Future<void> initialize({Uri? tokenEndpoint}) async {
+    var credentials = await _credentialsStorage.read();
```

Comment:
I dont like `var`s I would use getter for creating credential instance or using the one read from storage. that's just a suggestion.